features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/recorder/MappingContext.kt

```diff
@@ -18,6 +19,7 @@ import com.datadog.android.sessionreplay.utils.ImageWireframeHelper
  * @param imageWireframeHelper a helper tool to capture images within a View
  * @param textAndInputPrivacy the text and input privacy level to use when building the wireframes
  * @param imagePrivacy the image recording configuration to use when building the wireframes
+ * @param touchPrivacyManager the manager of handling touch privacy area.
```

Comment:
if I get the purpose right

```suggestion
 * @param touchPrivacyManager the manager to handle touch privacy area.
```

Reply:
updated